### DISCUSSION TITLE
Does doks support markdown image tag with a hyperlink source instead of local resource?

### DISCUSSION THREAD

**Discussion by vivekragunathan** [thuliteio/doks#584] [Q&A]
I see that image markdown for local resource works:

```markdown
![mountains!](/assets/images/san-juan-mountains.jpg "mountains")
```

but if the image source is not local 

```markdown
![hexagon](https://imgur.com/gallery/ArN48vy "hexagon")
```

... then rendering of the site breaks ...

![image](https://user-images.githubusercontent.com/292160/147843374-dc9631bc-e5c2-4e99-93d3-4f11e5be8d32.png)

Is this a known issue? If now, what am I missing?

**Comment by h-enk** [comment]
Hugo supports this as of [v0.91.0](https://github.com/gohugoio/hugo/releases/tag/v0.91.0)

So, I will be updating `./layouts/_default/_markup/render-image.html` for supporting this in the upcoming Doks release.

If you can't wait: replace line 1:

```html
{{ $image := .Page.Resources.GetMatch .Destination -}}
```

, with:

```
{{ $image := "" }}
{{ if (urls.Parse .Destination).IsAbs }}
  {{ $image = resources.GetRemote .Destination }}
{{ else }}
  {{ $image = .Page.Resources.GetMatch .Destination }}
{{ end }}
```

![Snag_4eeed34](https://user-images.githubusercontent.com/3902872/148087315-a507c0bd-fd9b-4bdc-b6f9-ce5362a77d22.png)

**Comment by vivekragunathan** [comment]
Is this released?

I wanted to know how to update doks, so went [**here**](https://getdoks.org/docs/help/how-to-update/). But the page talks about updating the dependent node modules. Where do I find how to update doks itself?

**Reply by h-enk** [reply]
Will be released soonish

**Reply by h-enk** [reply]
7123e6dfaee51d59064cb4b019c3d6a6ba39105d

**Reply by h-enk** [reply]
> I wanted to know how to update doks, so went here. But the page talks about updating the dependent node modules. Where do I find how to update doks itself?

The [Doks child theme](https://getdoks.org/docs/prologue/quick-start/#doks-child-theme) uses the [Doks npm package](https://www.npmjs.com/package/@hyas/doks), which you can update by running (see also [Update packages](https://getdoks.org/docs/help/how-to-update/#update-packages)):

```bash
npm update @hyas/doks
```
